snapcraft/parts/setup_assets.py

```diff
@@ -49,8 +49,13 @@ def setup_assets(
     gui_dir.mkdir(parents=True, exist_ok=True)
 
     _write_snap_directory(assets_dir=assets_dir, prime_dir=prime_dir, meta_dir=meta_dir)
-    _write_snapcraft_runner(prime_dir=prime_dir)
-    # TODO: write snapcraft
```

Comment:
why is this TODO gone? Are we writing snapcraft.yaml and manifest.yaml now?

Reply:
I believe it is a duplicate of the TODO on line 237:
`# TODO: record manifest and source snapcraft.yaml`